### PR TITLE
Tests: add regression test for redis URL with empty username

### PR DESCRIPTION
**Pull request by cryptomilk**
Covers the unix://:SECRET@/path.sock format (empty username, password only), which was missing from the existing test cases for PR #12239.

## Proposed change

add regression test for redis URL with empty username and password only

## Type of change

- [ ] Bug fix: non-breaking change which fixes an issue.
- [ ] New feature / Enhancement: non-breaking change which adds functionality. _Please read the important note above._
- [ ] Breaking change: fix or feature that would cause existing functionality to not work as expected.
- [ ] Documentation only.
- [x] Other. Please explain: Add regression test

## Checklist:

- [x] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [x] If applicable, I have included testing coverage for new code in this PR, for [backend](https://docs.paperless-ngx.com/development/#testing) and / or [front-end](https://docs.paperless-ngx.com/development/#testing-and-code-style) changes.
- [ ] If applicable, I have tested my code for breaking changes & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [x] If applicable, I have checked that all tests pass, see [documentation](https://docs.paperless-ngx.com/development/#back-end-development).
- [x] I have run all Git `pre-commit` hooks, see [documentation](https://docs.paperless-ngx.com/development/#code-formatting-with-pre-commit-hooks).
- [ ] I have made corresponding changes to the documentation as needed.
- [ ] In the description of the PR above I have disclosed the use of AI tools in the coding of this PR.

### CODE REVIEW

File: src/paperless/tests/settings/test_custom_parsers.py [hunk 1]
```diff
@@ -79,6 +79,16 @@ class TestRedisSocketConversion:
                 ),
                 id="celery_style_socket_with_credentials",
             ),
+            # Empty username, password only: unix://:SECRET@/path.sock
+            # Regression test for https://github.com/paperless-ngx/paperless-ngx/pull/12239
```
Comment (shamoon) [inline src/paperless/tests/settings/test_custom_parsers.py]:
```suggestion
```

Comment (cryptomilk) [reply src/paperless/tests/settings/test_custom_parsers.py]: I should remove that line?

Comment (cryptomilk) [reply src/paperless/tests/settings/test_custom_parsers.py]: The test a few lines above had that, so I just did a copy & paste...

**Review by shamoon — APPROVED** [review]
